src/views/NotFoundView.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import LoginView from "./LoginView";
+
+export const NotFoundView = () => (
+    <>
+        <h1 className="theme-text-light">Aby mieć dostęp do tych zaspobów, powinieneś być zalogowany.</h1>
```

Comment:
Poprawka językowa: literówka plus uniknięcie rodzaju gramatycznego.
```suggestion
        <h1 className="theme-text-light">Aby uzyskać dostęp do tych zasobów, zaloguj się.</h1>
```